res/layout/reaction_recipient_item.xml

```diff
@@ -0,0 +1,81 @@
+        android:focusable="true"
+        android:background="?attr/conversation_list_item_background"
+        android:paddingLeft="16dp"
+        android:paddingStart="16dp"
+        android:paddingRight="10dp"
+        android:paddingEnd="10dp">
+
+  <org.thoughtcrime.securesms.components.AvatarImageView
+      android:id="@+id/contact_photo_image"
+      android:layout_width="40dp"
+      android:layout_height="40dp"
+      android:foreground="@drawable/contact_photo_background"
+      android:cropToPadding="true"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      tools:src="@color/blue_600"
+      android:contentDescription="@null" />
+
+    <LinearLayout android:layout_width="0dp"
+                  android:layout_height="wrap_content"
+                  android:layout_weight="1"
+                  android:layout_marginLeft="11dp"
+                  android:layout_marginStart="11dp"
+                  android:paddingRight="16dp"
+                  android:paddingEnd="16dp"
+                  android:orientation="vertical">
+
+        <org.thoughtcrime.securesms.components.emoji.EmojiTextView
+            android:id="@+id/name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:drawablePadding="5dp"
+            android:ellipsize="marquee"
+            android:singleLine="true"
+            android:fontFamily="sans-serif"
+            android:textSize="16sp"
+            tools:text="Frieeeeeeedrich Nieeeeeeeeeetzsche" />
+
+        <LinearLayout android:id="@+id/addr_container"
+                      android:orientation="horizontal"
+                      android:layout_width="wrap_content"
+                      android:layout_height="wrap_content">
+
+            <TextView android:id="@+id/addr"
+                      android:layout_width="wrap_content"
+                      android:layout_height="wrap_content"
+                      android:textDirection="ltr"
+                      android:singleLine="true"
+                      android:ellipsize="marquee"
+                      android:textAppearance="?android:attr/textAppearanceSmall"
+                      android:textSize="14sp"
+                      android:fontFamily="sans-serif-light"
+                      tools:text="user@example.com" />
+
+        </LinearLayout>
+
+    </LinearLayout>
+
+    <org.thoughtcrime.securesms.components.emoji.EmojiTextView
+        android:id="@+id/reaction"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:textSize="22dp"
+        android:fontFamily="sans-serif"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text=":)" />
```

Comment:
the padding right to the reaction should be the same as left of the avatar

to me, it looks too small